static/js/compose_actions.js

```diff
@@ -1,3 +1,5 @@
+var autosize = require('autosize/dist/autosize.js');
```

Comment:
These should just be `require('autosize')`.

Reply:
I tried that now, but it doesn't seem to be working. The webpage just doesn't load if I use 
`var autosize = require('autosize')`.

Reply:
That’s weird, it works fine for me. Did you get any webpack errors or browser console errors?